operator/bundle/manifests/loki-operator.clusterserviceversion.yaml

```diff
@@ -581,7 +581,14 @@ spec:
           - create
         serviceAccountName: default
       deployments:
-      - name: loki-operator-controller-manager
+      - label:
+          app.kubernetes.io/instance: loki-operator-v0.0.1
+          app.kubernetes.io/managed-by: operator-lifecycle-manager
+          app.kubernetes.io/name: loki-operator
+          app.kubernetes.io/part-of: cluster-logging
+          app.kubernetes.io/version: 0.0.1
+          control-plane: controller-manager
+        name: loki-operator-controller-manager
```

Comment:
Do we want do have the labels back here cc @xperimental ?

Reply:
These labels should be fine as they are the labels on the Deploment itself and not the selector used for finding the pods (a few lines below). I had wondered why the Deployment had so few labels on my PR related to the selector, so this is better than before from my point of view.